scripts/jwt_issuer/up.sh

```diff
@@ -23,5 +24,5 @@ jwt-issuer-up
 
 jq -n '{
     JWT_ISSUER_WORKSPACE: env.JWT_ISSUER_WORKSPACE,
-    JWT_ISSUER: env.JWT_ISSUER,
```

Comment:
Why have we removed JWT_ISSUER? The idea of this part is anything we've exported and is therefore important to future execution is printed here in case the environment can't be persisted easily, namely in the system tests

Reply:
added it back

Reply:
I still don't see it?

Reply:
fixed